spec/requests/checkouts_spec.rb

```diff
@@ -118,4 +118,52 @@
       expect(response).to redirect_to(checkouts_url)
     end
   end
+
+  describe "PATCH /return" do
+    it "updates the return time of requested checkout" do
+      patch return_checkout_url(checkout)
+      checkout.reload
+      expect(checkout.return_time).to be <= DateTime.current
+    end
+    it "redirects to the checkouts list" do
+      patch return_checkout_url(checkout)
+      expect(response).to redirect_to(checkouts_url)
+    end
+  end
+
+  describe "PATCH /return_all" do
+    before :each do
+      FactoryBot.reload
```

Comment:
Looking into it further, I shouldn't be. It apparently reloads the factory definitions, rather than clearing previously entered FactoryBot objects. Which I should have known it wasn't doing because the created user was still active after calling it and isn't necessary anyways since FactoryBot overrides itself if really necessary...Will remove it.

Reply:
We have DatabaseCleaner setup so that it should be wiping objects after each test.